release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -67,7 +67,8 @@ Red Hat to assist in the deployment process, or create your own. You are also
 free to create the required resources through other methods; the ARM templates
 are just an example.
 
-//See [link] for details.
+See xref:../installing/installing_azure/installing-azure-user-infra.adoc#installing-azure-user-infra[Installing a cluster on Azure using ARM templates]
```

Comment:
Would we want to define _ARM_? I'm thinking the CPU architecture, but that can't be correct in this instance.

Reply:
It was actually defined earlier in that entry; it didn't show up as new in this PR since I only added the new link. It's for "Azure Resource Manager".